packages/core/chrome/core-chrome-browser-internal/src/ui/header/collapsible_nav.scss

```diff
@@ -24,3 +24,19 @@
     text-decoration: underline; /* 2 */
   }
 }
+
+/**
+ * Allows the CollapsibleNavGroups to be viewed on
+ * very small screen sizes and when the browser Zoom is high
+ */
+@media (max-height: 15em) {
```

Comment:
Let's pull this out to a variable so that 1. the intention of it is clear and 2. we can reuse it across different selectors if needed

```scss
$screenHeightBreakpoint: 15em;

@media (max-height: $screenHeightBreakpoint) {
```